CONTRIBUTING.md

```diff
@@ -1,5 +1,16 @@
 # Contribution guidelines
 
+## Startup phase
+
+The RSQKit is still under heavy development.
+This means structure and content may still change a lot.
+During this phase, we organize contributions through EVERSE internal documents and communication channels.
+At some point we will likely move towards a more GitHub-centered contribution workflow, as outlined below.
+
+Specifically, the main "location" for assigning authors to contributions is the spreadsheet.
```

Comment:
I think the spreadsheet in question is not linked here

Reply:
It's not meant to be publicly accessible but just for member of EVERSE at the moment during this phase (the plan is to make it public but for bootup we are just doing things internally) - it's a google doc at the moment - but if we want to put a link then perhaps we should move the document to the EVERSE ownCloud.

Happy to hear what others think

Reply:
I wrote it a bit obscurely on purpose, indeed not meant to have a link :) We should remove this startup phase disclaimer as soon as possible (when the phase is over), but right now it's meant to clarify the process for our main contributors: EVERSE members.